region/brinstar/kraid.json

```diff
@@ -1631,6 +1691,44 @@
                   "note": "Wait to hit one of Mini-Kraid's spikes, then run through."
                 }
               ]
+            },
+            {
+              "id": 2,
+              "strats": [
+                {
+                  "name": "Green Pirates and Mini-Kraid Farm",
+                  "notable": false,
+                  "requires": [
+                    {"resetRoom": {
+                      "nodes": [1, 2],
+                      "mustStayPut": false
+                    }},
+                    {"or": [
+                      "Super",
+                      "Ice",
+                      "Wave",
+                      "Spazer",
+                      "Plasma",
+                      {"and": [
+                        "Bombs",
+                        "canCarefulJump"
+                      ]}
+                    ]},
+                    {"refill": ["Energy", "Missile", "Super"]}
```

Comment:
I think the pirate farm should be separate.  And you are mainly gaining missiles there.

Reply:
The reason for combining them is so that we could include the option to use Supers (farmed from Mini-Kraid) to kill the Pirates

Reply:
On second thought, it'd be organized to have two separate farms, but that's not actually a good reason to.  A couple things are simpler with them grouped.

Reply:
Cool, and this can always be refined later if you have a different idea for how to split up the strats. I suppose Mini-Kraid could still be a useful energy farm even with no way to kill the Pirates, and that isn't expressed with the combined strat here